src/Symfony/Component/Security/Http/Authenticator/JsonLoginAuthenticator.php

```diff
@@ -131,6 +143,11 @@ public function isInteractive(): bool
         return true;
     }
 
+    public function setTranslator(?TranslatorInterface $translator)
```

Comment:
if we make the DI config `ignoreOnInvalid`, we can remove nullable here.

Reply:
Good idea :)